packages/edit-site/src/components/page-pages/index.js

```diff
@@ -129,13 +135,17 @@ export default function PagePages() {
 						</a>
 					);
 				},
+				type: 'set',
```

Comment:
In terms of API, I think we could just use `type`, which would be useful for a few field operations: filtering and cell editing, for example.

- The filter uses `type: set` and the `setList` to generate the UI for the filter and the set of valid values.

<img width="459" alt="Captura de ecrã 2023-10-13, às 13 45 00" src="https://github.com/WordPress/gutenberg/assets/583546/9cbbf8b1-1091-4511-a9fc-3cd1ad8a081d">

- When editing the cell, the same information could be leveraged to show a dropdown.

Reply:
A field may have different filters (see current example with `IN` and `NOT IN` filters), but those match to different REST API parameters, so it's not enough to declare a `type`. We actually need to map specific types of filters to the corresponding REST API parameter.